src/Presentation/TwigEnvironmentConfigurator.php

```diff
@@ -61,6 +61,7 @@ public function getEnvironment( Twig_Environment $twig, array $loaders, array $e
 
 	private function setDefaultTwigVariables( Twig_Environment $twig ): void {
 		$twig->addGlobal( 'basepath', $this->config['web-basepath'] );
+		$twig->addGlobal( 'appconfig', $this->config );
```

Comment:
I have a bad feeling about passing the whole application configuration (with all the database and payment provider passwords) into the template layer. While it should be secure enough (we're the only ones that touch the templates), I created `setDefaultTwigVariables` especially for this purpose of passing only needed variables.